web/src/settings_realm_user_settings_defaults.js

```diff
@@ -67,25 +62,6 @@ export function set_up() {
         });
     });
 
-    if (!page_params.development_environment) {
-        $("#realm_dense_mode").on("change", (e) => {
-            const val = $(e.target).prop("checked");
-            if (val) {
-                $container.find(".information-density-settings").hide();
-                return;
-            }
-
-            if (
-                !realm_user_settings_defaults.dense_mode &&
-                (realm_user_settings_defaults.web_font_size_px !== NON_COMPACT_MODE_FONT_SIZE_PX ||
-                    realm_user_settings_defaults.web_line_height_percent !==
-                        NON_COMPACT_MODE_LINE_HEIGHT_PERCENT)
-            ) {
-                $container.find(".information-density-settings").show();
-            }
-        });
-    }
```

Comment:
I am not convinced this code is actually dead; `realm_dense_mode` might just be a dynamically constructed selector.
Seems to correspond to "Compact mode" here. I confirmed it existed via `$("#realm_dense_mode")` in the JS console. 

![image](https://github.com/user-attachments/assets/633b6be8-175d-4c42-aa57-09fa641aef93)